src/neptune/metadata_containers/metadata_container.py

```diff
@@ -450,8 +452,8 @@ def stop(self, *, seconds: Optional[Union[float, int]] = None) -> None:
         self._op_processor.stop(sec_left)
 
         if self._mode not in {Mode.OFFLINE, Mode.DEBUG}:
-            logger.info("Explore the metadata in the Neptune app:")
-            logger.info(self.get_url().rstrip("/") + "/metadata")
+            metadata_url = self.get_url().rstrip("/") + "/metadata"
+            logger.info(f"Explore the metadata in the Neptune app: {metadata_url}")
```

Comment:
@normandy7 That's for you to review :wink:

Reply:
Looks good!